### PR TITLE
Update AnalysisIcepak.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/application/AnalysisIcepak.py [hunk 1]
```diff
@@ -90,26 +124,27 @@ def apply_icepak_settings(self, ambienttemp=20, gravityDir=5, perform_minimal_va
...
-        type :
-            type of the property. "Boundary", "Excitation", "Setup", "Mesh" (Default value = None)
-        objectname :
-            param property:
-        property :
-            
+        objectname : str
+            Name of the object.
+        property : str
+            Name of the design proerty.
```
Comment (PipKat) [inline pyaedt/application/AnalysisIcepak.py:134]:
```suggestion
            Name of the design property.
```

File: pyaedt/application/AnalysisIcepak.py [hunk 2]
```diff
@@ -46,27 +78,29 @@ def mesh(self):
...
-            Ambient Temperature. It can be an integer or a parameter already created in AEDT (Default value = 20)
-        gravityDir :
-            Gravity direction index [0, 5] (Default value = 5)
-        perform_minimal_val :
-            True Perform Minimal validation. False perform Full Validation (Default value = True)
-        default_fluid :
-            default Fluid ."Air"
+        ambienttemp : int, optional
+            Ambient temperature, which can be an integer or a parameter already 
+            created in AEDT. The default is 20.)
```
Comment (akaszynski) [inline pyaedt/application/AnalysisIcepak.py:87]:
```suggestion
            created in AEDT. The default is 20.
```

File: pyaedt/application/AnalysisIcepak.py [hunk 3]
```diff
@@ -46,27 +78,29 @@ def mesh(self):
...
+        gravityDir : int, optional
+            Gravity direction index [0, 5]. The default is ``5``.
```
Comment (akaszynski) [inline pyaedt/application/AnalysisIcepak.py:89]:
```suggestion
            Gravity direction index in the range ``[0, 5]``. The default is ``5``.
```

File: pyaedt/application/AnalysisIcepak.py [hunk 4]
```diff
@@ -90,26 +124,27 @@ def apply_icepak_settings(self, ambienttemp=20, gravityDir=5, perform_minimal_va
...
-            property value
-            
+            Value of the property.
             
-            :Example:
+        Example
+        -------
             
-            val = ipk.get_property_value('BoundarySetup:Source1', 'Total Power')
+        >>>val = ipk.get_property_value('BoundarySetup:Source1', 'Total Power')
```
Comment (akaszynski) [inline pyaedt/application/AnalysisIcepak.py:147]:
```suggestion
        >>> val = ipk.get_property_value('BoundarySetup:Source1', 'Total Power')
```

File: pyaedt/application/AnalysisIcepak.py [hunk 5]
```diff
@@ -146,23 +181,25 @@ def get_property_value(self, objectname, property, type=None):
...
-        design :
-            starting application object (examples hfss1= HFSS3DLayout)
-        object_list :
-            List of object to copy (Default value = None)
-        no_vacuum : bool
-            define if vacuum objects have to be copied (Default value = True)
-        no_pec : bool
-            define if pec objects have to be copied (Default value = True)
+        design : str
+            Starting application object. For example, ``hfss1= HFSS3DLayout``.
```
Comment (akaszynski) [inline pyaedt/application/AnalysisIcepak.py:189]:
```suggestion
            Starting application object. For example, ``hfss1=HFSS3DLayout``.
```

File: pyaedt/application/AnalysisIcepak.py [hunk 6]
```diff
@@ -235,21 +273,23 @@ def assignmaterial(self, obj, mat):
...
-            
+           Name of the property.
 
         Returns
         -------
+        bool
+            ``True`` when successful, ``False`` when failed.
 
-        """
+"""
```
Comment (akaszynski) [inline pyaedt/application/AnalysisIcepak.py:292]:
```suggestion
        """
```

**Review by akaszynski — APPROVED** [review]
Approved with some suggestions.

**Pull request merged**